src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -187,6 +187,21 @@
             Ref("VariableAssignmentSegment"),
         ]
     ),
+    ArithmeticBinaryOperatorGrammar=OneOf(
+        Ref("PlusSegment"),
+        Ref("MinusSegment"),
+        Ref("DivideSegment"),
+        Ref("MultiplySegment"),
+        Ref("ModuloSegment"),
+        Ref("BitwiseAndSegment"),
+        Ref("BitwiseOrSegment"),
+        Ref("BitwiseXorSegment"),
+        Ref("BitwiseLShiftSegment"),
+        Ref("BitwiseRShiftSegment"),
+        # Add MySQL's DIV and MOD segment
+        Ref("DivOperatorSegment"),
+        Ref("ModOperatorSegment"),
+    ),
```

Comment:
Can it be rewritten as `get_grammar` ... `copy` like the grammars above it?

Reply:
It will be good 👍